lib/modules/apostrophe-express/index.js

```diff
@@ -491,12 +506,12 @@ module.exports = {
         }
         // Reset the cookie so that if its lifetime somehow detaches from
         // that of the session cookie we're still OK
-        res.cookie(self.apos.csrfCookieName, token);
+        res.cookie(self.apos.csrfCookieName, token, (self.options.csrf && self.options.csrf.cookie) || {});
```

Comment:
should this also be checking for `self.options.csrf.cookie.secure` per your comments above?

Reply:
No, `secure` is only one example of the properties you can give to `cookie`. Our job just to get `cookie` to `res.cookie` as a third argument, they might add `secure: true` or some other thing or nothing and that's not our biz

Reply:
@boutell ah, implementation example. noted! good to go ✅